modules/aggregation-layers/src/gpu-grid-layer/gpu-grid-layer.js

```diff
@@ -187,7 +187,7 @@ export default class GPUGridLayer extends CompositeLayer {
 
     // return props to the sublayer constructor
     return super.getSubLayerProps({
-      id: 'grid-cell',
+      id: 'gpu-grid-cell',
```

Comment:
Instead of overriding the `getSubLayerProps()` method, pass the props directly to `SubLayerClass`:

```js
return new SubLayerClass(
  defaultSubLayerProps,
  this.getSubLayerProps({id, updateTriggers}),
  nonOverridableProps
);
```

It is 1) more performant and 2) allows for user override with the [_subLayerProps prop](https://deck.gl/#/documentation/deckgl-api-reference/layers/composite-layer?section=_sublayerprops-object-experimental).

See https://github.com/uber/deck.gl/blob/master/modules/aggregation-layers/src/hexagon-layer/hexagon-layer.js#L356

Reply:
updated.